ios/browser/svg/BUILD.gn

```diff
@@ -57,6 +57,9 @@ source_set("svg") {
   sources += skia_svg_public
   sources += skia_svg_sources
 
+  sources -=
+      [ "//third_party/skia/modules/svg/src/SkSVGOpenTypeSVGDecoder.cpp" ]
+
```

Comment:
Not sure if related but may want to keep an eye on https://github.com/brave/brave-core/pull/22434/files when rebasing